src/main/java/org/folio/circulation/resources/RequestCollectionResource.java

```diff
@@ -124,6 +126,7 @@ void replace(RoutingContext routingContext) {
 
     write(representation, "id", getRequestId(routingContext));
 
+    final ItemRepository itemRepository = new ItemRepository(clients, true, true, true);
```

Comment:
here also may use var instead of ItemRepository

Reply:
done